aws-redshift-clusterparametergroup/src/main/java/software/amazon/redshift/clusterparametergroup/UpdateHandler.java

```diff
@@ -36,47 +44,110 @@ protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
+                        .makeServiceCall(this::readTags)
+                        .handleError(this::operateTagsErrorHandler)
+                        .done((tagsRequest, tagsResponse, client, model, context) -> ProgressEvent.<ResourceModel, CallbackContext>builder()
+                                .callbackContext(callbackContext)
+                                .callbackDelaySeconds(0)
+                                .resourceModel(Translator.translateFromReadTagsResponse(tagsResponse))
+                                .status(OperationStatus.IN_PROGRESS)
+                                .build()))
 
-                .then(progress ->
-                        proxy.initiate("AWS-Redshift-EventSubscription::Update::UpdateTags", proxyClient, progress.getResourceModel(), progress.getCallbackContext())
-                                .translateToServiceRequest(resourceModel -> Translator.translateToUpdateTagsRequest(request.getDesiredResourceState(), resourceModel, resourceName))
-                                .makeServiceCall(this::updateTags)
-                                .handleError(this::operateTagsErrorHandler)
-                                .done((tagsRequest, tagsResponse, client, model, context) -> ProgressEvent.<ResourceModel, CallbackContext>builder()
-                                        .callbackContext(callbackContext)
-                                        .callbackDelaySeconds(0)
-                                        .resourceModel(request.getDesiredResourceState())
-                                        .status(OperationStatus.IN_PROGRESS)
-                                        .build()))
+                .then(progress -> proxy.initiate("AWS-Redshift-EventSubscription::Update::UpdateTags", proxyClient, progress.getResourceModel(), progress.getCallbackContext())
+                        .translateToServiceRequest(resourceModel -> Translator.translateToUpdateTagsRequest(request.getDesiredResourceState(), resourceModel, resourceName))
+                        .makeServiceCall(this::updateTags)
+                        .handleError(this::operateTagsErrorHandler)
+                        .done((tagsRequest, tagsResponse, client, model, context) -> ProgressEvent.<ResourceModel, CallbackContext>builder()
+                                .callbackContext(callbackContext)
+                                .callbackDelaySeconds(0)
+                                .resourceModel(request.getDesiredResourceState())
+                                .status(OperationStatus.IN_PROGRESS)
+                                .build()))
 
-                .then(progress -> proxy.initiate("AWS-Redshift-ClusterParameterGroup::Update::ResetInstance", proxyClient, progress.getResourceModel(), progress.getCallbackContext())
+                .then(progress -> proxy.initiate("AWS-RedshiftServerless-Workgroup::Update::ReadParameters", proxyClient, progress.getResourceModel(), progress.getCallbackContext())
```

Comment:
is this similar to calling read handler ?

Reply:
yes, but only read parameters not the parameter group instance